src/txn/txn.c

```diff
@@ -702,6 +709,13 @@ __txn_commit_timestamps_assert(WT_SESSION_IMPL *session)
 	if (!F_ISSET(txn, WT_TXN_TS_COMMIT_KEYS | WT_TXN_TS_DURABLE_KEYS))
 		return (0);
 
+	cursors = NULL;
+	if (txn->mod_count != 0) {
+		WT_RET(__wt_calloc(
+		    session, sizeof(WT_CURSOR *), txn->mod_count, cursor_buf));
```

Comment:
Is "sizeof(WT_CURSOR *)" correct here. Should it be "sizeof(WR_CUROSR **)". I know a pointer of any kind would be of the same size so this is purely semantics I think.

Reply:
I think it's ok but I had to double check when I wrote this so maybe I'm wrong. I want an array of `WT_CURSOR *`. So the size is `sizeof(WT_CURSOR *)` and the number of elements is `txn->mod_count`.